src/modm/communication/amnb/node.hpp

```diff
@@ -314,7 +327,8 @@ class Node : public modm::Resumable<6>
 	}
 	response_status{ResponseStatus::NotWaiting};
 
-	static constexpr uint8_t PRIORITY_BITS{5};
+	static constexpr uint8_t MIN_TX_TRIES{20};
+	static constexpr uint8_t PRIORITY_BITS{6};
 	static constexpr uint8_t RESCHEDULE_MASK_SHORT{7};
 	static constexpr uint8_t RESCHEDULE_MASK_LONG{11};
```

Comment:
I'm not happy about the magic numbers here, but I don't have a proper test setup here and currently lack instrumentation of this code. This isn't even true p-persistent CSMA, and the timings depend on the baudrate and median message size. I'm going to have to do some more research once I have time.